src/lms-harmonizer/edfi_lms_harmonizer/scripts/postgresql/0012_view_missing_assignment_category_descriptors.sql

```diff
@@ -0,0 +1,12 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+create view lmsx.missing_assignment_category_descriptors as
+    select * from lms.assignment lmsassignment
+	where not exists(
+			select 1 from edfi.descriptor assignmentcatdescriptor where
+				assignmentcatdescriptor.codevalue = lmsassignment.assignmentcategory
```

Comment:
We didn't think to put an index on `lmsassignment.assignmentcategory`. We might want to do that someday.